test/theaters.js

```diff
@@ -50,7 +37,42 @@ test('get theaters from lat/long', function (assert) {
   api = new Showtimes('45.531531531531535,-122.61220863200342')
   api.getTheaters(function (err, theaters) {
     assert.equal(err, null)
-    assert.ok(theaters.length > 1)
+    assert.ok(theaters.length > 1, 'more than one theater')
+    assert.end()
+  })
+})
+
+test('get theaters from lat/long and later get filtered theaters for first theater name', function (assert) {
+  api = new Showtimes('45.531531531531535,-122.61220863200342')
+  api.getTheaters(function (err, theaters) {
+    assert.equal(err, null)
+    api.getTheaters(function (err2, theaters2) {
```

Comment:
No, it should not. It's ok, what I want to test here is that the result when filtering include the first theater :)